src/LibraryManager.Vsix/MultilingualResources/Microsoft.Web.LibraryManager.Vsix.pt-BR.xlf

```diff
@@ -52,19 +52,19 @@
 This may require an internet connection.
 
 Do you want to continue?</source>
-          <target state="translated">Será instalado um pacote NuGet para aumentar o processo MSBuild, mas nenhum arquivo será adicionado ao projeto.
-
-Isso pode exigir uma conexão de internet.
-
-Você quer continuar?</target>
+          <target state="translated">Será instalado um pacote NuGet para aumentar o processo MSBuild, mas nenhum arquivo será adicionado ao projeto. Isso pode exigir uma conexão com a internet. Deseja continuar?</target>
```

Comment:
Can we use another word instead of "aumentar"? Maybe "complementar" ?

Reply:
We checked your feedback and implemented all changes.
Next PR will contain corrected translation.